data/MSCP/config.json

```diff
@@ -1,26 +1,25 @@
 {
-  "resourceID": "0x0000000000000000000005c22ba65f65adffadfc0947382f2e7c286a0fe45505",
+  "resourceID": "0x000000000000000000000681f946225048505d7222c06cbdf850b1f0d9a6b505",
```

Comment:
"resourceID": "0x0000000000000000000005c22ba65f65adffadfc0947382f2e7c286a0fe45505",
this reward id is correct. keep this.